dice_ml/explainer_interfaces/dice_pytorch.py

```diff
@@ -376,8 +376,8 @@ def stop_loop(self, itr, loss_diff):
         """Determines the stopping condition for gradient descent."""
 
         # intermediate projections
-        if((self.project_iter > 0) & (itr > 0)):
-            if((itr % self.project_iter) == 0):
+        if self.project_iter > 0 and itr > 0:
```

Comment:
why are all these unrelated changes part of this PR? I see a lot of formatting changes for if.

Reply:
I was trying to fix all if (X) to if X at once and that's why it all ended up in the same PR. Shifting it to a different PR